### DISCUSSION TITLE
I've designed a bunch of 16px icons for a project recently, should I add them to the /img/ folder here?

### DISCUSSION THREAD

**Discussion by Gaai** [sbrin/lopaka#84] [Show and tell]
Just discovered this project and it looks like it's gonna be a real time saver. Very nice work! Hopefully you can add C style formatting of the functions too, so we can directly use the code with esp-idf too. Perhaps I can be of some help there. But never know with the time I've got. 

Anyway. I've got icons for battery life, wifi and network signals, phone (calls), volume, microphone, music, play, warning, error, sd card etc. Let me know if you're interested. I might make more in the near future. 
Some examples:
<img width="12" alt="battery_50_16x24" src="https://github.com/sbrin/lopaka/assets/59128156/c8523b03-4505-4de0-8914-2c348777df99">
<img width="12" alt="battery_charging_16x24" src="https://github.com/sbrin/lopaka/assets/59128156/e3f6b6e7-0e19-486b-8be1-dd5db4a12176">
<img width="7" alt="bluetooth_16x14" src="https://github.com/sbrin/lopaka/assets/59128156/d4ad4578-c0ae-4d8a-bad2-4d73e759948e">
<img width="8" alt="network_3_bars_16x15" src="https://github.com/sbrin/lopaka/assets/59128156/b2897596-af7f-4d4d-a790-a610bd2e4655">
<img width="8" alt="notification_bell_16x16" src="https://github.com/sbrin/lopaka/assets/59128156/ea2e97c7-758c-4550-bf8a-9b5221bf801d">
<img width="9" alt="volume_no_sound_16x18" src="https://github.com/sbrin/lopaka/assets/59128156/28680d97-27c6-41d8-9147-58b6dcea1d98">
<img width="10" alt="wifi_50_16x19" src="https://github.com/sbrin/lopaka/assets/59128156/84a97be1-c04e-488e-9d09-b53f430a2e03">

**Comment by sbrin** [comment]
@Gaai Thank you. These icons look awesome!

Can you share all you've got in a zip archive? I'll be happy to add them

**Reply by sbrin** [reply]
@Gaai Thank you. By the way, would you mind if I ask you a few questions regarding your projects? I also can answer your questions regarding Lopaka features and maybe help on solving your issues. How can I contact you? Please drop me a line if you are interested to milin@lopaka.app

**Reply by Gaai** [reply]
Send you an email.
